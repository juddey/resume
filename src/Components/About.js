import React from 'react'
import Typography from '@material-ui/core/Typography'

const About = () => (
  <Typography
    style={{ maxWidth: '550px', color: 'black', textAlign: 'center' }}
  >
    Reskilled in web and mobile development from a ERP software implementation
    career, I'm looking for remote-based short to mid term contracts. Rates
    negotiable.
  </Typography>
)
export default About
