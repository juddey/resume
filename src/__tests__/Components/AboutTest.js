import React from 'react'
import {
  createComponentWithIntl,
  shallowWithIntl
} from '../../../src/Support/Utils'
import About from '../../Components/About'

const div = document.createElement('div')
const wrapper = shallowWithIntl(<About />)

test('component exists', () => {
  expect(wrapper.length).toBe(1)
})

test('component structure', () => {
  expect(wrapper.name()).toEqual('WithStyles(Typography)')
  expect(wrapper.children().length).toEqual(1)
})

test('renders to snapshot', () => {
  const tree = createComponentWithIntl(<About />, div).toJSON()
  expect(tree).toMatchSnapshot()
})
