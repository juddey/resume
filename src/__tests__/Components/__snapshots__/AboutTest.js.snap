// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to snapshot 1`] = `
<p
  className="MuiTypography-root-1 MuiTypography-body1-10"
  style={
    Object {
      "color": "black",
      "maxWidth": "550px",
      "textAlign": "center",
    }
  }
>
  Reskilled in web and mobile development from a ERP software implementation career, I'm looking for remote-based short to mid term contracts. Rates negotiable.
</p>
`;
